src/components/Dialog/DialogTitle/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import useStyles from './styles';
+import Text, {ITextProps} from '../../Text';
+
+interface IDialogTitle extends ITextProps {}
+
+const DialogTitle: React.FC<IDialogTitle> = ({children, style, ...rest}) => {
+  const styles = useStyles();
+  return (
+    <Text style={[styles.dialogTitle, style]} variant="h4" {...rest}>
```

Comment:
what if we make children: string | ReactNode?

Reply:
It doesn't really make sense to do so. If someone would need custom component he can use his component instead `Dialog.Title`. It's not mandatory to use `Dialog.Title` inside `Dialog`